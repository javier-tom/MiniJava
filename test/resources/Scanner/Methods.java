public int a (int b, boolean c, int[] d) {
    int e = b + d[0];
    return e;
}

public int b(int a) {
    return a;
}