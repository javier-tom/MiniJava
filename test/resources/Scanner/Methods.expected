PUBLIC INT ID(a) LPAREN INT ID(b) COMMA BOOLEAN ID(c) COMMA INT LSQUARE RSQUARE ID(d) RPAREN LCURLY INT ID(e) BECOMES ID(b) PLUS ID(d) LSQUARE INTEGER_LITERAL(0) RSQUARE SEMICOLON RETURN ID(e) SEMICOLON RCURLY PUBLIC INT ID(b) LPAREN INT ID(a) RPAREN LCURLY RETURN ID(a) SEMICOLON RCURLY